test/StripeResource.spec.js

```diff
@@ -32,4 +35,245 @@ describe('StripeResource', function() {
       expect(headers).to.not.include.keys('Stripe-Version');
     });
   });
+
+  describe('Retry Network Requests', function() {
+    // Use a real instance of stripe as we're mocking the http.request responses.
+    var realStripe = require('../lib/stripe')(utils.getUserStripeKey());
+    var options = {
+      host: stripe.getConstant('DEFAULT_HOST'),
+      path: '/v1/charges',
+      data: {
+        amount: 1000,
+        currency: 'usd',
+        source: 'tok_visa',
+        description: 'test'
+      },
+      params: 'amount=1000&currency=usd&source=tok_visa&description=test'
+    };
+
+    afterEach(function() {
```

Comment:
should this be `beforeEach`?

Reply:
This is more of a clean up step rather than set up, so I opted for `afterEach`.